### ISSUE TITLE
npm install

### ISSUE THREAD

**Issue by dcharbonnier** [minio/minio-js#613]
we should be able to install from a git url but the project is not setup correctly and the dist files are not build during the setup...

https://github.com/FormidableLabs/radium/issues/731

**Comment by krishnasrinivas** [comment]
Tinkered a bit to fix this but unable to get devDependencies installed and compile the es6 sources. Will look at this later.

**Comment by deekoder** [comment]
@dcharbonnier  we will address this in future. Moving it to future milestone.

**Comment by gilvaju** [comment]
How should I use it then?

Clone the repository anywhere and only install globally? And then use 'require'?

**Comment by gilvaju** [comment]
![image](https://user-images.githubusercontent.com/677039/36188691-d596d926-1134-11e8-8db9-c4eb3cef3cd4.png)
I've tried to use it in any way I know, but it always gives the same error

**Comment by krishnasrinivas** [comment]
@gilvaju you dont want to use `npm install minio`? If you want to use from master you can do:
git clone https://github.com/minio/minio-js/
cd minio-js
npm install && npm run compile

and `var minio = require("./minio-js")`

**Comment by gilvaju** [comment]
npm install minio does not work @krishnasrinivas

had made the clone, but had not compiled, I'll test

**Comment by krishnasrinivas** [comment]
@gilvaju note that you cant do `npm install minio` inside minio-js repo. You have to do `npm install minio` in another repo where you want to use `minio-js` npm module.

```
krishna@escape:/tmp$ npm install minio
npm WARN saveError ENOENT: no such file or directory, open '/tmp/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/tmp/package.json'
npm WARN tmp No description
npm WARN tmp No repository field.
npm WARN tmp No README data
npm WARN tmp No license field.

+ minio@4.0.0
added 34 packages in 0.923s
krishna@escape:/tmp$ ls node_modules/minio/
appveyor.yml  CONTRIBUTING.md  dist  docs  examples  gulpfile.js  LICENSE  MAINTAINERS.md  package.json  README.md  README_zh_CN.md
krishna@escape:/tmp$ 
```

**Comment by gilvaju** [comment]
@krishnasrinivas 

I see, that's what I'm doing. I'm trying to use it on a vues project.

However, he is having problems in resovering his dependencies, I believe it is for the reason that issue

**Comment by krishnasrinivas** [comment]
@gilvaju I see the screenshot you have provided. It is incorrect to write HTML/Javascript like that. Can you please open a different issue? Because your issue is not related the issue #613

**Comment by gilvaju** [comment]
I am using Vuejs, single file components.
I believe I could not run the clone because I did not run the compile
I'll give you feedback as soon as I test
In any case, wait to use via npm

**Comment by harshavardhana** [comment]
Fixed by #700
